src/daemon/daemon.cpp

```diff
@@ -1877,6 +1877,12 @@ try // clang-format on
         return status_promise->set_value(status);
     }
 
+    status = cmd_vms(instances, [this](auto& vm) {
+        init_mounts(vm.vm_name);
+
+        return grpc::Status::OK;
+    });
+
```

Comment:
Do we need another check on `status` here and set the `status_promise` accordingly?